### ISSUE TITLE
Accessibility - Links

### ISSUE THREAD

**Issue by mcmullengreg** [XavierUniversity/V4#15]
New content styles should accommodate accessible links. Links should also maintain a descriptive nature. Existing issues include:
- **WCAG A**
  -  Non-distinguishable links - 3,390
  - Image link has no alternative text - 1,221
  - Link text is not sufficient - 46
  - Link identified by only color - 4
  - Links should be combined - 1
  - Link refers to a non-existent element - 2,959
- **WCAG AA**
  - None
- **WCAG AAA**
  -  Non-distinguishable links - 3,390
  - Link text is non descriptive - 173
